src/content/components/PriorityGuide/PriorityGuide.scss

```diff
@@ -23,6 +23,10 @@
   padding: 0 3px;
   border-radius: 3px;
   font-weight: bold;
+
+  &:hover {
+    cursor: pointer;
+  }
```

Comment:
You don't need to put this under `:hover`. It can go with the other properties above. The `cursor` property can only ever do anything if the cursor is over the element (in other words, when the element is hovered)